component/src/main/java/org/wso2/carbon/identity/authenticator/bitly/BitlyAuthenticator.java

```diff
@@ -263,4 +273,43 @@ protected Map<ClaimMapping, String> getSubjectAttributes(OAuthClientResponse tok
         }
         return claims;
     }
+
+    @Override
+    protected String sendRequest(String url, String accessToken) throws IOException {
+
+        if (log.isDebugEnabled()) {
+            log.debug("URL: " + url);
```

Comment:
Shall we mention "URL"  as user info url of bitly?

Reply:
Fixed